### ISSUE TITLE
Initialization using previously anotated data

### ISSUE THREAD

**Issue by pedrolagomarsino** [flatironinstitute/CaImAn#647]
1. Operating system: Linux 
 
 2. Python version: 3.7
 
 3. Working environment: Jupyter Notebook/Python IDE
 
  6. CaImAn installation process: conda-forge  

- **If you're not reporting an error, type your message below**
Hi, as suggested in the paper "In some cases, good initializations are available" (the example given being the dual labeling of neurons), in my case I have a series of putative centers of neurons (or containing boxes) and would like to initialize the algorithm using such information instead of the greedy_roi default or any other of the available methods. Is this possible? 
Thanks

**Comment by epnev** [comment]
@pedrolagomarsino There is a notebook demo `demo_seeded_CNMF` that shows an example of how to do this. The demo assumes that you have the exact spatial extent of the neurons, but I expect it should also work with square bounding boxes. Let me know if you have any questions.

**Comment by pedrolagomarsino** [comment]
Hi!

Thanks for the help! I do have a question.
I did what's suggested in the demo_seeded_CNMF. That is, I created N
individual masks (one for each rectangular bounding box) and put them into
a matrix Ain.

Ain = np.zeros((512*512, boxes.shape[0]),dtype=bool)for i in range(len(boxes)):
    frame = np.zeros((512,512))
    frame[boxes[i,0].astype('int'):boxes[i,2].astype('int'),boxes[i,1].astype('int'):boxes[i,3].astype('int')]=1
    Ain[:,i] = frame.reshape(512*512).astype(bool)

Where boxes is a numpy.matrix with the coordinates of each box ( boxes[i,:]
= [xmin,ymin,xmax,ymax] ).
This works smoothly in the sense that finds the cell spatial and temporal
components in each box. However it ONLY finds the cells where there is a
mask, If I add only 10 masks it will find only 10 cells.
I thought this was a way of initializing the algorithm, instead of a
constrain on where to look for cells. Am I doing something wrong or is this
the correct behavior?

If this is what its supposed to do, is there a way of initializing the
algorithms whith the seeds but allow it to look for more (or less) cells?
Thanks again!

Pedro
